bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
actually, could we add Shared / Owned type aliases, and then use different type classes for .as[Shared] vs .as[Owned] or is scala too smart for that?